request.go

```diff
@@ -69,23 +78,25 @@ func prepareAuthRequest(requestID string, username string, password string) (req
 	req.Args = make(map[string]interface{})
 	req.Args["sasl"] = base64.StdEncoding.EncodeToString(simpleAuth)
 
-	return
+	return req, nil
 }
 
 // formatMessage takes a request type and formats it into being able to be delivered to Gremlin Server
-func packageRequest(req request) (msg []byte, err error) {
+func packageRequest(req request) ([]byte, error) {
 	j, err := json.Marshal(req) // Formats request into byte format
 	if err != nil {
-		return
+		return nil, err
 	}
-	mimeType := []byte("application/vnd.gremlin-v2.0+json")
-	msg = append([]byte{0x21}, mimeType...) //0x21 is the fixed length of mimeType in hex
+	mimetypelen := byte(len(MimeType))
```

Comment:
Does not look like a valid variable name

Reply:
fixed